aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -75,10 +79,10 @@ module AptosFramework::Stake {
 
     /// Withdraw from active delegation, it's moved to pending_inactive if locked_until_secs < current_time or
     /// directly deposit if it's not from an active validator.
-    fun withdraw_active(account: &signer, from: address) acquires ValidatorInfo, ValidatorSet {
+    fun withdraw_active(account: &signer, from: address) acquires StakePool, ValidatorSet {
         let addr = Signer::address_of(account);
         let current_time = Timestamp::now_seconds();
-        let stake_pool = &mut borrow_global_mut<ValidatorInfo>(from).stake_pool;
+        let stake_pool = borrow_global_mut<StakePool>(from);
```

Comment:
should we provide some error code when not allowed to withdraw?

Reply:
yeah, I'll deal with error code later, currently it's all 0